src/explorer/SubscriptionTreeItem.ts

```diff
@@ -125,6 +125,10 @@ export class SubscriptionTreeItem extends SubscriptionTreeItemBase {
         try {
             // enable HTTP logs by default
             await newSite.enableHttpLogs();
+            if (wizardContext.newSiteOS === WebsiteOS.windows) {
+                //enable Application logs by defaul (only for windows)
+                await newSite.enableApplicationLogs();
```

Comment:
Rather than make two methods, just add a flag to `enableHttpLogs` so that we only have to call `updateLogsConfig` (which actually sends the request) once